cmd/runner/testcase.go

```diff
@@ -173,20 +173,31 @@ func (t *testCaseDC) run(client endpoint, server endpoint) error {
 	cmd.Stdout = &cmdOut
 	cmd.Stderr = &cmdOut
 
-	err := cmd.Start()
+	testLog, err := os.Create(filepath.Join(testOutputsDir, "test.txt"))
```

Comment:
The standard way to log things in go is via the `log.Logger` API. E.g.: https://yourbasic.org/golang/log-to-file/

Reply:
The contents of this file aren't log output -- it's the tuples written on line 183.

Reply:
It's possible to write lines to a `log.Logger` without prefixing it with a timestamp. You would just do `logger.SetFlags(0)`. You could then simplify line 183 a bit. Instead of:
```go
_, _ = testLog.WriteString(fmt.Sprintf("%s,%s,%s,%s", client.name, server.name, t.name, "error"))
```
you would o
```go
logger.Printf("%s,%s,%s,error", client.name, server.name, t.name)
```

Reply:
I hear you, but this isn't a `log`. It's just a file that holds four strings. We could just rename `testLog` to something else (`dumbFileHandle`) to make that more clear?

Reply:
I think that would be better.